moveit_core/collision_detection/src/allvalid/collision_env_allvalid.cpp

```diff
@@ -46,7 +46,7 @@ namespace
 {
 rclcpp::Logger getLogger()
 {
-  return moveit::getLogger("collision_detection.world_allvalid");
+  return moveit::getLogger("moveit.core.collision_detection.world_allvalid");
```

Comment:
```suggestion
  return moveit::getLogger("moveit.core.collision_detection.env_allvalid");
```
Should we maybe rename this to match the file name?